src/App.tsx

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
The `classnames` library is required for defining classes conditionally, as per the checklist. Please integrate this into your code .